src/zenml/zen_stores/rest_zen_store.py

```diff
@@ -470,6 +452,23 @@ def _initialize_database(self) -> None:
         """Initialize the database."""
         # don't do anything for a REST store
 
+    def _create_default_stack(
+        self,
+        workspace_id: UUID,
+    ) -> StackResponse:
+        workspace = self.get_workspace(workspace_id)
+
+        raise RuntimeError(
+            f"Unable to create default stack in workspace "
+            f"{workspace.name}."
+        )
+
+    def _create_default_workspace(self) -> WorkspaceResponse:
+        raise RuntimeError(
+            f"Unable to create default workspace "
+            f"{self._default_workspace_name}."
+        )
+
```

Comment:
Instead of using an abstract approach, maybe you could only call these in the SQL zen store DB initialization to begin with.

Reply:
I've moved all the database initialization into the SQLZenStore as you suggested, much better :)